package.json

```diff
@@ -18,15 +18,15 @@
     "lint": "eslint .",
     "test": "node test.js",
     "posttest": "npm run lint",
-    "mocha": "./node_modules/.bin/_mocha --timeout 40000 --require strong-mocha-interfaces --require test/init.js --require test/*.test.js --ui strong-bdd"
+    "mocha": "./node_modules/.bin/_mocha --timeout 40000 --require strong-mocha-interfaces --require test/init.js --ui strong-bdd"
   },
   "dependencies": {
     "async": "^1.5.0",
     "cloudant": "^1.4.0",
     "debug": "^2.2.0",
     "lodash": "^4.17.4",
     "loopback-connector": "^4.0.0",
-    "loopback-connector-couchdb2": "^1.0.0",
+    "loopback-connector-couchdb2": "https://github.com/strongloop/loopback-connector-couchdb2.git#recover-skipped-test",
```

Comment:
This needs to be pointed to the latest patch once the [Couch PR](https://github.com/strongloop/loopback-connector-couchdb2/pull/31) lands.